fe/src/components/layouts/Header/Header.tsx

```diff
@@ -0,0 +1,95 @@
+import {
+  Box,
+  Flex,
+  Heading,
+  HStack,
+  Link,
+  IconButton,
+  useDisclosure,
+  useColorModeValue,
+  Stack,
+} from '@chakra-ui/react';
+import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
+
+interface NavigationItem {
+  id: number,
+  label: string;
+  href: string;
+}
+
+const NAV_ITEMS: Array<NavigationItem> = [
```

Comment:
This is all nice and well but a few tips:
1. You can let ts infer the types. My philosophy is to create a interface/type only when you need to use it more than once.
2. The `key` prop for React maps can be any unique string/int so you don't have to define it, but can use the `href` instead. Or just the index from the map

I propose:
```diff
@@ -11,20 +11,12 @@ import {
 } from '@chakra-ui/react';
 import { HamburgerIcon, CloseIcon } from '@chakra-ui/icons';
 
-interface NavigationItem {
-  id: number,
-  label: string;
-  href: string;
-}
-
-const NAV_ITEMS: Array<NavigationItem> = [
+const NAV_ITEMS = [
   {
-    id: 0,
     label: 'Home',
     href: '/',
   },
   {
-    id: 1,
     label: 'About',
     href: '/about',
   },
@@ -52,7 +44,7 @@ export default function Header(): JSX.Element {
           >
             {NAV_ITEMS.map(navItem => (
               <Link
-                key={navItem.id}
+                key={navItem.href}
                 px={2}
                 py={1}
                 rounded="md"
@@ -74,7 +66,7 @@ export default function Header(): JSX.Element {
           <Stack as="nav" spacing={4}>
             {NAV_ITEMS.map(navItem => (
               <Link
-                key={navItem.id}
+                key={navItem.href}
                 px={2}
                 py={1}
                 rounded="md"

```

Reply:
Good ideas. 👍